app/Nova/User.php

```diff
@@ -272,6 +275,12 @@ public function lenses(Request $request)
     public function actions(Request $request)
     {
         return [
+            (new Actions\SyncAccess)
+                ->canSee(function ($request) {
+                    return true;
+                })->canRun(function ($request, $user) {
+                    return true;
```

Comment:
Is this meant to be available to everyone? I know in theory it is harmless to run at arbitrary times, but it could be potentially confusing to users.

Reply:
It is intentionally visible to anyone with Nova access.

I guess only admins would need to run it though. Will fix.